test/cluster/test_refresh.py

```
@@ -172,3 +174,100 @@ async def test_refresh_deletes_uploaded_sstables(manager: ManagerClient):
             await wait_for_upload_dir_empty(upload_dir)
 
         shutil.rmtree(tmpbackup)
+
+
+async def wait_for_row_count_on_host(cql, host, ks, table, expected, timeout=120):
+    '''Wait until `table` reports `expected` rows when read through `host`.
+
+    View updates are generated asynchronously, off the staging sstables, so the
+    count has to be polled rather than read once.
+    '''
+    stmt = SimpleStatement(f"SELECT COUNT(*) FROM {ks}.{table}", consistency_level=ConsistencyLevel.LOCAL_ONE)
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)